docs/api/web-contents.md

```diff
@@ -532,6 +532,24 @@ Emitted when `webContents` wants to do basic auth.
 
 The usage is the same with [the `login` event of `app`](app.md#event-login).
 
+#### Event: 'dialog'
+
+Returns:
+
+* `event` Event
+* `dialogDetails` [MessageDialogDetails](structures/message-dialog-details.md) | [PromptDialogDetails](structures/prompt-dialog-details.md)
+* `callback` Function
+  * `success` boolean - Indicates whether the dialog was closed successfully (OK) or not (Cancel).
+  * `result` string - The response string when accepting a prompt dialog.
+
+Emitted when `webContents` wants to show javascript dialog alert, confirm or prompt.
+The type of a particular dialog can be checked with the `dialogDetails.type` property.
+You can use `dialogDetails.frameProcessId` and `dialogDetails.frameRoutingId` to construct a `webFrame` by calling `webFrameMain.fromId()`.
+
+Calling `event.preventDefault()` indicates that the dialog request is handled by the user.
+If you call `event.preventDefault()` make sure to call `callback()` afterwards.
+The renderer will be blocked until then.
```

Comment:
It's not clear to me from these docs what the effect of calling or not calling `preventDefault` is. What happens if I don't call `preventDefault`? What is the "default action" here?

Reply:
I clarified the default behavior a bit. Let me know if that makes sense to you now.